### ISSUE TITLE
[feature request] Add the ability for rehearsal graph to graph multiple directories at once

### ISSUE THREAD

**Issue by robostheimer** [rehearsal-js/rehearsal-js#1217]
Given that I have a mono-repo and I want to get a graph of all the different internal packages inside of my repo, it would be nice to be able to run rehearsal graph once and have it give me output files for more than one directory given a parent directory.

Example call might look like: `rehearsal graph packages --subdirectories,app --subdirectories`.

A note, I have a gist set up to run locally that accomplishes this and may be useful in grokking the ask here.

https://gist.github.com/robostheimer/9b5b6324e530fde35f2989515ba211a8

**Comment by lynchbomb** [comment]
Yeah totally we can bake something like this https://gist.github.com/lynchbomb/84115ee2f11731dbb2fa9aa08d681191 into the graph command for monorepos
